vega_sim/api/data.py

```diff
@@ -7,6 +7,7 @@
 import vega_sim.grpc.client as vac
 import vega_sim.proto.data_node.api.v2 as data_node_protos_v2
 import vega_sim.proto.vega as vega_protos
+import vega_sim.proto.vega.events.v1.events_pb2 as events_protos
```

Comment:
I am just wondering whether this is the v1 apis that are getting deprecated, or do I have things mixed up?

Reply:
I thought it was just v1 datanode APIs that are getting deprecated, as far as I know there are no v2 APIs for the other gRPC methods..?